inventory-playbook.yml

```diff
@@ -395,6 +395,30 @@
         https_port: "{{ custom_setup_info.https_port }}"
       when: custom_setup_info.https_port != ''
 
+    - name: Definição de nova senha aleatória para postgres
+      ansible.builtin.set_fact:
+        postgres_new_pass: "{{ lookup('community.general.random_string', length=16, special=False) }}"
+        postgres_password_was_updated: false
+
+    - name: Substituição da senha do postgres, caso esteja com a senha padrão 'postgres'
+      block:
+        - name: Alteração da senha na database
+          community.postgresql.postgresql_user:
+            db: postgres
+            login_host: "{{ shrink_inventory.postgres.address }}"
+            login_user: "{{ shrink_inventory.postgres.default_user }}"
+            login_password: "{{ shrink_inventory.postgres.default_password }}"
+            name: "{{ shrink_inventory.postgres.default_user }}"
+            password: "{{ postgres_new_pass }}"
+          register: db_password
+
+        - name: Definição da variável para permitir atualização
+          ansible.builtin.set_fact:
+            postgres_password_was_updated: true
+          when: db_password.changed
+
+      when: postgres_new_pass is defined and shrink_inventory.postgres.address is defined and shrink_inventory.postgres.default_password is defined and shrink_inventory.postgres.default_user is defined and shrink_inventory.postgres.default_password == 'postgres'
```

Comment:
when: shrink_inventory.postgres.default_password is defined and shrink_inventory.postgres.default_password == 'postgres'